src/test/java/com/emc/mongoose/storage/driver/pravega/kvs/util/docker/PravegaNodeContainer.java

```diff
@@ -0,0 +1,59 @@
+package com.emc.mongoose.storage.driver.pravega.kvs.util.docker;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.exception.NotFoundException;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.core.DockerClientBuilder;
+import com.github.dockerjava.core.command.PullImageResultCallback;
+
+import java.io.Closeable;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+public class PravegaNodeContainer
+				implements Closeable {
+
+	public static final int PORT = 9090;
+	private static final Logger LOG = Logger.getLogger(PravegaNodeContainer.class.getSimpleName());
+	private static final String IMAGE_NAME = "pravega/pravega:0.3.2";
```

Comment:
If the test has been moved from Storage Driver Pravega, I assume the image should be changed in each of the drivers since `pravega/pravega:0.3.2` is 2 years old now.

Reply:
Sure, this test isn't currently used though,